website/src/pages/jobs.tsx

```diff
@@ -310,12 +298,29 @@ const CareersPage = props => {
                                     You strive to continuously grow and learn by genuinely soliciting feedback early and
                                     often, and humbly reflecting on your past mistakes.
                                 </p>
-                                <img src="../careers/7growth.svg" />
+                                <img src="/careers/7growth.svg" />
                             </div>
                         </div>
                     </Carousel.Item>
                 </Carousel>
 
+                <Carousel
+                    className="bg-white height-800 height-sm-800 pt-md-0 pt-1"
+                    prevIcon={<ArrowLeftIcon color="#808080" />}
+                    nextIcon={<ArrowRightIcon color="#808080" />}
+                >
+                    {new Array(6).fill(null).map((el, i) => (
+                        <Carousel.Item key={`review-${i}`}>
+                            <div className="d-flex align-center height-750 height-sm-650">
+                                <img
+                                    src={`/careers/review${i + 1}.png`}
+                                    className="d-block m-auto col-11 col-lg-5 mw-lg-600 mw-sm-500"
```

Comment:
You could probably get away with `mw-600`. It would solve the issue of the images underlapping slide bullet point UI and the images from not stretching too far on wider screens. And then you could remove those spacing styles in this PR since we'll be porting over the new spacings 🙂

Reply:
I need `col-11 mw-sm-500 mw-600` for the mobile sizing though. Oh yea I'll port over your spacing changes now 🌟